Command/BatchConsumerCommand.php

```diff
@@ -16,6 +16,9 @@ final class BatchConsumerCommand extends BaseRabbitMqCommand
      */
     protected $consumer;
 
+    /** @var string */
```

Comment:
I think it should be
```php
/**
 * @var int
 */
```

Property above use multiline phpdoc and property always contain an integer

Reply:
Fixed. Thank you.